### PR TITLE
Updated documentation around s3 buckets

### PR DESCRIPTION
**Pull request by jvasallo**
- Mentioned you can also provide a list of bucket names

### CODE REVIEW

File: _docs/configuration_files/pipeline_json/services.rest [hunk 1]
```diff
@@ -40,9 +40,9 @@ Add Lambda access.
 ``s3``
 ******
 
-Add S3 access.
+Add S3 access. You may need to override default templates, see :ref:`templates_path`. Alternatively, you can rovide a list of s3 bucket names to be added to the instance profile.
```
Comment (ndcampbell) [inline _docs/configuration_files/pipeline_json/services.rest]: typo `rovide`, i assume you mean `provide`

File: _docs/configuration_files/foremast_config.rst [hunk 2]
```diff
@@ -88,6 +88,8 @@ FQDN Of your spinnaker Gate instance. This is where all API calls to Spinnaker w
 
     | *Required*: Yes
 
+.. _templates_path:
```
Comment (ndcampbell) [inline _docs/configuration_files/foremast_config.rst]: What is this addition for?

Comment (jvasallo) [inline _docs/configuration_files/foremast_config.rst]: To link to the template_path if they choose to override our defaults we ship with.

File: _docs/configuration_files/pipeline_json/services.rest (hunk 1 above)
Comment (e4r7hbug) [inline _docs/configuration_files/pipeline_json/services.rest]: Haha! Got you Joel!

Comment (jvasallo) [inline _docs/configuration_files/pipeline_json/services.rest]: fixed!

**Pull request merged**